package.json

```diff
@@ -11,9 +11,14 @@
   "scripts": {
     "build": "ember build",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "ember test",
+    "build-storybook": "ember build && build-storybook -p 9001 -s dist",
+    "storybook": "ember serve & start-storybook -p 9001 -s dist"
   },
   "dependencies": {
+    "@storybook/addon-actions": "^5.0.11",
+    "babel-loader": "^8.0.6",
+    "babel-plugin-ember-modules-api-polyfill": "^2.9.0",
```

Comment:
`@storybook/addon-actions` and `babel-loader` should be moved to devDependencies. `babel-plugin-ember-modules-api-polyfill` shouldn't be necessary